corehq/apps/reports/standard/monitoring.py

```diff
@@ -1355,7 +1355,7 @@ def _group_cell(self, group_id, group_name):
         url = self._make_url(base_url, params)
 
         return util.format_datatables_data(
-            self._html_anchor_tag(url, group_name),
+            self._html_anchor_tag(url, group_name.encode('utf-8')),
```

Comment:
@millerdev I think that would also work. If you think thats a better way to move forward I can make a new PR with that instead.